modules/raycast/raycast_occlusion_cull.cpp

```diff
@@ -108,8 +108,12 @@ void RaycastOcclusionCull::RaycastHZBuffer::update_camera_rays(const Transform3D
 
 	debug_tex_range = td.z_far;
 
+#ifdef THREADS_ENABLED
```

Comment:
It'd be more maintainable that the `WorkerThreadPool` acts as a dumb sequential runner of the tasks in cases like this, so there's no need to add these disjunctives every time.

Reply:
I removed the `THREADS_ENABLED` here, as it was finally superfluous.